### ISSUE TITLE
Support for native timestamp and uint32 types

### ISSUE THREAD

**Issue by andrea-dintino** [dyne/Zenroom#189]
Script: 

```
Rule unknown ignore
rule check version 1.0.0
Scenario ecdh: saso

Given I have a 'number' named 'timestamp'
Then print the 'timestamp'
```
data:

`{ "timestamp": 1619178116640 }`

Output:

[*] Direct Zencode execution
 .  Rule unknown ignore
 .  Zencode version match: 1.0.0+46ba100
 .  rule check version 1.0.0
{"timestamp":1.619178e+12}

**Comment by jaromil** [comment]
Rather than rely on Lua's number type, we need a native "zenroom.uint" of 32bit length. In case of use for timestamps this should cover future dates up to year 2106. In case past dates are needed, then "zenroom.int" would support years from 1970 to 2038.

An extensible approach would be "zenroom.timestamp" to be a special case, accepting epochs for now and later extend support for more date formats.

We will continue this analysis off channel and provide a solution ASAP

**Comment by jaromil** [comment]
solved in 35b59ad5d39a9299361721bcfe5a90a9d9015f7c
